### PR TITLE
Second "ontology" ag-Grid

### PR DESCRIPTION
**Pull request by ivansg44**
image

### CODE REVIEW

File: geem/static/geem/js/geem_validation.js [hunk 1]
```diff
@@ -56,3 +67,16 @@ function update_grid(grid_options, data) {
...
+
+
+/**
+ * Update ontology grid headers.
+ * Must be called after a new form is rendered.
+ * @param grid_options - Ontology validation grid options
+ */
+function update_ontology_grid(grid_options) {
+	const column_defs = top.form.components.map(function(component) {
+		return {headerName: component.label, field: component.label}
```
Comment (ddooley) [inline geem/static/geem/js/geem_validation.js]: Might want field to be component.id ?

File: geem/static/geem/js/geem_form.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ function OntologyForm(domId, resource, settings, callback) {
 			$('li[role="menuitem"][data-ontology-id="' + self.entityId + '"]').addClass('active')
 
 			var entity = get_form_specification_component(self.entityId)
+			top.form.components = entity.components
```
Comment (ddooley) [inline geem/static/geem/js/geem_form.js]: So the callback param above is a name of a function that will be called after form spec entity is built and all form rendering is done.  In form_portal.js line 111 there is a call to OntologyForm() that contains the function ref. to portal_entity_form_callback, so you could add "top.form.components = form.components" there?

File: geem/static/geem/js/geem_validation.js (hunk 1 above)
Comment (ivansg44) [reply geem/static/geem/js/geem_validation.js]: Done

File: geem/static/geem/js/geem_form.js (hunk 2 above)
Comment (ivansg44) [reply geem/static/geem/js/geem_form.js]: Done. I moved the call to update the ontology grid there too, so the ontology grid code is less spread out.

**Pull request merged**